api/src/main/java/org/openmrs/module/queue/api/QueueEntryService.java

```diff
@@ -9,6 +9,9 @@
  */
 package org.openmrs.module.queue.api;
 
+import static org.openmrs.util.PrivilegeConstants.ADD_VISITS;
```

Comment:
Do we need to import static?

Reply:
Thought it was better than using canonical name in the `@Authorized` tag.